lib/config/marshal/select.go

```diff
@@ -119,7 +118,7 @@ func (fm FileMarshallers) UnmarshalDefault(path string, data []byte, def Marshal
 func (fm FileMarshallers) UnmarshalFilePrefix(prefix string, value interface{}) (string, error) {
 	var errs []error
 	for _, candidate := range fm {
```

Comment:
Linux uses `/` while Windows uses `\`. Use the `filepath` library to sanitize file paths regardless of OS.

Reply:
Both the docs and the code for `ToSlash` says it does the opposite - it replaces the OS-specific slash character with a forward slash - https://cs.opensource.google/go/go/+/refs/tags/go1.21.4:src/path/filepath/path.go;l=216

Reply:
`/` is what `openssh` expects on windows. For example, when I do `scp` to copy a file to a Windows machine, I need to use `/` in the destination path. I am not sure if `sanitize` is the correct word. Perhaps `reformat` or `standardize` are more accurate terms.

Reply:
Oh, that makes sense! I thought you meant sanitize meaning the other direction - unknown Unix-like path into OS-specific path

Meta note - these review annotations you're making will get lost once the review is merged - if they're important, they should become code comments. For example, if you think a future editor will thing that `ToSlash()` is a bug, you can put a justification comment like `// openssh expects forward-slash-delimited paths on all platforms; convert paths to forward slashes if needed`

Reply:
Ok, I will put the comment call-outs as code comments.